### ISSUE TITLE
[Feature Request] Add llucy support.

### ISSUE THREAD

**Issue by gergesh** [TeamNewPipe/NewPipe#1358]
[llucy](https://llucy.net/) is an alluc successor, a service that searches for videos in various streaming sites.

I can try implementing this myself (though I haven't done any real android development before so it may take a while or just straight-up fail) but was wondering, given the legal-but-shady nature of the site, if that's a service that'd be welcome as an addition to NewPipe.

**Comment by theScrabi** [comment]
Does this require login?

**Comment by theScrabi** [comment]
But yes please give it a shot :)

**Comment by gergesh** [comment]
Yes, it does require login. A free account will get you 200 results per day and (as I just found out) is limited to 1000 all-time. I can still try implementing it, but not sure if it's worth it at this point.
